staticd/static_zebra.c

```diff
@@ -314,9 +314,11 @@ void static_zebra_nht_register(struct static_nexthop *nh, bool reg)
 		DEBUGD(&static_dbg_route,
```

Comment:
Let's fix this debug messaging here.  We should have a `registering address for nexthop resolution` and a `already registered address for nexthop resolution` debug paths.  So that we can actually debug this when something goes wrong in the future

Reply:
OK

Reply:
@donaldsharp looks addressed now